### ISSUE TITLE
Start and stop polling

### ISSUE THREAD

**Issue by nikolalsvk** [nikolalsvk/render_async#78]
Add options to be able to start polling. Let's say you want to start polling after the user clicks a button and you emit an event "render-async-start-polling", it would be cool to do that.

Also, have render_async stop polling when you emit a certain event, e.g. "stop-polling-now".

Idea rose from this comment https://github.com/renderedtext/render_async/issues/67#issuecomment-490540519

**Comment by weavermedia** [comment]
Stop polling option will be great! 🎉

**Comment by nikolalsvk** [comment]
Hey y'all, I've released [version 2.1.1](https://rubygems.org/gems/render_async/versions/2.1.1) which solves this issue!

If you have time, please try it out and see if it brings any bugs with it, thanks 🍰

**Comment by pivotal-pmital** [comment]
@nikolalsvk I am not seeing how to use this magical feature that you are referring to in version 2.1.1. The PR that is associated with this issue is also closed. Can you help me understand how to use it? Previously I was using this gem at commit 9d9e8edce8e7ab6ba9a169ef1fe70257ca6aacba but the feature seems to be gone now. Can you help me out with this?? Thanks!

**Comment by nikolalsvk** [comment]
I'm really sorry about this @pivotal-pmital. I don't see the commit you've mentioned in the master commits, it never got merged actually :(

Anyways, you can start and stop polling using **toggle** feature of the gem. I should have made this more clear in the Polling section of the README. If you take a look at [Toggle section](https://github.com/renderedtext/render_async#toggle-event) you will see this:

Also, you can mix interval and toggle features. This way, you can turn polling on, and off by clicking the "Detail" button. In order to do this, you need to pass toggle and interval arguments to render_async call like this:

```erb
<a href='#' id='detail-button'>Detail</a>
<%= render_async comments_path, toggle: { selector: '#detail-button', event: :click }, interval: 2000 %>
```

### TLDR

You can pass in interval option together with toggle and start and stop polling that way.
